tests/performance/locust_files/platform_ping_test.py

```diff
@@ -20,25 +19,16 @@
 assert locust_plugins  # nosec
 
 
-load_dotenv()  # take environment variables from .env
-
-_UNDEFINED = "undefined"
-
-
-class LocustAuth(NamedTuple):
-    username: str = os.environ.get("SC_USER_NAME", _UNDEFINED)
-    password: str = os.environ.get("SC_PASSWORD", _UNDEFINED)
-
-    def defined(self) -> bool:
-        return _UNDEFINED not in (self.username, self.password)
+class LocustAuth(BaseSettings):
+    SC_USER_NAME: str = Field(default=..., examples=["<your username>"])
```

Comment:
Get used to use `SecretStr`...  and then `_auth.SC_USER_NAME.get_secret_value()`, `_auth.SC_PASSWORD.get_secret_value()`

Reply:
Good point. Done